packages/admin-ui-docs/content/table/code.mdx

```diff
@@ -0,0 +1,739 @@
+        id: 2,
+        name: 'Color 2',
+        image:
+          'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRsvxLXJ3jW2hQox375iMAcaScYMpUmXk1dFw&usqp=CAU',
+        price: 30,
+      },
+      {
+        id: 3,
+        name: 'Color 3',
+        image:
+          'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcR0FyiZM7bDPsDEMtg0Zs2HXNwe2xbVh55IZA&usqp=CAU',
+        price: 10,
+      },
+    ]
+
+  const state = useTableState({
+    columns: [
+        {
+          id: 'image',
+          header: 'Image',
+          resolver: {
+            type: 'image',
+            preview: {
+              display: true,
+              size: 'regular',
+              delay: 0,
+            },
+          },
+        },
+        {
+          id: 'name',
+          header: 'Name',
+          resolver: {
+            type: 'text',
+            isNameCell: true,
+            mapText: (item) => item.name,
+          },
+        },
+        {
+          id: 'price',
+          header: 'Price',
+        },
+      ],
+    items: fruits,
+     onRowClick: (item) => alert(`Row clicked: ${item.name}`),
+  })
+
+  return <Table state={state} />
+}
+```
+
+### Sorting
+
+To use the base sorting configuration, that matches the majority of use cases, you just need to pass the `compare` function to the columns that you want to sort by. Two params are accepted, representing two items - you must return a boolean that proves their equality.
+
+```ts
+type Compare = (a: T, b: T) => boolean
+```
+
+The following example allows ordering by `name`, `lastSale` and `price`.
+
+```jsx noInline live
+const items = Array(3)
+  .fill()
+  .map((_, id) => {
+    return {
+      id: `${id}`,
+      name: faker.commerce.productName(),
+      lastSale: faker.date.past().toDateString(),
+      price: faker.commerce.price(),
+    }
+  })
+
+function CompareExample() {
+  const state = useTableState({
+    columns: [
+      {
+        id: 'name',
+        header: 'Product Name',
+        compare: (a, b) => b.name.localeCompare(a.name),
+      },
+      {
+        id: 'lastSale',
+        header: 'Last Sale',
+        compare: (a, b) => {
+          const aLastSale = new Date(a.lastSale).valueOf()
+          const bLastSale = new Date(b.lastSale).valueOf()
+
+          return bLastSale - aLastSale
+        },
+      },
+      {
+        id: 'price',
+        header: 'Price',
+        resolver: {
+          type: 'currency',
+          locale: 'en-US',
+          currency: 'USD',
+        },
+        compare: (a, b) => parseInt(b.price, 10) - parseInt(a.price, 10),
+      },
+    ],
+    items,
+    sort: {
+      initialValue: { order: 'ASC' },
+    },
+  })
+
+  return <Table state={state} />
+}
+
+render(<CompareExample />)
+```
+
+#### Configuration
+
+By using the `sort` property within `useTableState` you can configure the sorting to match specific use cases.
+
+##### initialValue
+
+Defines the table initial sorting value. `{ order?: 'ASC' | 'DSC', by?: string }`
```

Comment:
It just came to mind: I think I've never seen ASC and DSC being used before, but I've seen plenty of ASC and DESC (mainly from SQL), and it occurs to me that this might be a pattern for other APIs as well (to call ascending and descending order ASC and DESC, respectively).